src/microbe_stage/MicrobeInternalCalculations.cs

```diff
@@ -33,7 +33,8 @@ public static Vector3 MaximumSpeedDirection(IEnumerable<OrganelleTemplate> organ
 
     public static Vector3 GetOrganelleDirection(OrganelleTemplate organelle)
     {
-        return (Hex.AxialToCartesian(new Hex(0, 0)) - Hex.AxialToCartesian(organelle.Position)).Normalized();
+        // TODO: calculate this for organelles other than flagella
+        return MovementComponent.CalculateForce(organelle.Position, organelle.Definition.Components.Movement!.Momentum);
```

Comment:
I don't really like how the *force* is used here. I don't actually see how this can even fix the hex layout being offset, unless this is meant to fix something else...

Also a static helper method somewhere that is not as flagella specific and one that doesn't use momentum, should be done.